### ISSUE TITLE
Client | CMS Pages

### ISSUE THREAD

**Issue by jmaicaaan** [UltraRangers/gauntlet#33]
This would be the parent ticket of all the new CMS features we will have.
Let's have a branch of `feature/cms`, so for the other sub-tickets we will just branch off here.

With our current pages available, I think what we can do the with the following states:

- Make the login page as startup page (currently it is the "dashboard" with lists, buttons)
- Then redirect the page to the dashboard after logging in

**Comment by john20xdoe** [comment]
In terms of creating parents and sub-tasks, (or epic and substories), I think we can make use of Github's https://github.com/UltraRangers/gauntlet/projects tab, though I haven't tried it fully if it is enough to organize the tasks like JIRA does, or like the checklists in Trello.

**Comment by jmaicaaan** [comment]
Yeah we can use it but I think it is not that good compared to zenhub's board add on if you can look at it 😊

**Comment by john20xdoe** [comment]
I see, zenhub IS better. Nice find :+1:
